urllib3/contrib/pyopenssl.py

```diff
@@ -127,6 +129,17 @@ def extract_from_urllib3():
     util.ssl_.IS_PYOPENSSL = False
 
 
+def _validate_dependencies_met():
+    """
+    Verifies that PyOpenSSL's package-level dependencies have been met.
+    Throws `ImportError` if they are not met.
+    """
+    # Method added in `cryptography==1.1`; not available in older versions
+    from cryptography.x509.extensions import Extensions
+    if not hasattr(Extensions, "get_extension_for_class"):
+        raise ImportError("'cryptography' module missing required function (try upgrading the module?): 'cryptography.x509.extensions.Extensions.get_extension_for_class'")
```

Comment:
Regarding your other comments, thanks!  I'll update my PR accordingly the next time I have a chance to sit down at a computer and make the changes.